web/src/app/pes-admin/provider/provider.component.ts

```diff
@@ -41,21 +44,25 @@ export class ProviderComponent implements OnInit {
       this.setupFormEditProvider()
     }
   }
-
-  onBack(): void {
+  backToTheListProviders(): void {
```

Comment:
Renombrar a BackToProvidersList. En nombres de funciones no sulen ir los articulos que no suman expresividad (the/de a...etc)